### ISSUE TITLE
Existing Input for suggest or select.

### ISSUE THREAD

**Issue by Gytree** [palantir/blueprint#3152]
#### Environment

"react": "^16.6.0",
"react-dom": "^16.6.0",
"@blueprintjs/core": "^3.8.0",
 "@blueprintjs/select": "^3.2.1",

#### Question

How i can attach a suggest, select or popover components to a existing input?

**Comment by giladgray** [comment]
@Gytree this is not the intended usage for `Suggest` (it renders its own input that you can control via `inputProps`) but is trivial with the others:

`<Select ...><your input /></Select>`
`<Popover content={...}><your input /></Popover>`

**Comment by Gytree** [comment]
hi @giladgray I think I did not explain myself well. the input it's not rendered with react. it's an dom node that was added for other library. but i need to attach the suggest component to it. so i can't use the previous examples.

 i use the **document.getElementById** to get the element in my componentDidMount and i try to render the suggest with the reference to the element,but i have not found a way to do it.

any advice?
